modules/programs/zsh.nix

```diff
@@ -252,6 +253,13 @@ in
         default = {};
         description = "Options to configure oh-my-zsh.";
       };
+
+      localVariables = mkOption {
+        type = types.attrs;
+        default = {};
+        example = { POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=["dir" "vcs"]; };
+        description = "Extra local variables defined at the top of <filename>.zshrc</filename>";
```

Comment:
Can put the description in a '' string, also descriptions should end in a period:
```nix
description = ''
  Extra local variables defined at the top of <filename>.zshrc</filename>.
'';
```

Reply:
Done